npm/janus.d.ts

```diff
@@ -193,12 +241,14 @@ declare namespace JanusJS {
 
 		constructor(options: ConstructorOptions);
 
-		getServer(): string;
-		isConnected(): boolean;
-		getSessionId(): string;
 		attach(options: PluginOptions): void;
-		reconnect(options: ReconnectOptions): void;
-		destroy(options: DestroyOptions): void;
```

Comment:
any reason this bunch of declarations has been removed?

Reply:
mhhhh.... "stupid developer"? :-D